agent/Dockerfile

```diff
@@ -19,9 +19,9 @@ RUN mkdir -p agent \
     && chmod -R +r /var/lib/automation/config \
     && rm agent/mongodb-agent.tar.gz \
     && rm -r mongodb-mms-automation-agent-*
-
 RUN mkdir -p /var/lib/mongodb-mms-automation/probes/ \
-    && curl --retry 3 https://readinessprobe.s3-us-west-1.amazonaws.com/readiness -o /var/lib/mongodb-mms-automation/probes/readinessprobe \
+#    && curl --retry 3 https://readinessprobe.s3-us-west-1.amazonaws.com/readiness -o /var/lib/mongodb-mms-automation/probes/readinessprobe \
+    && curl --retry 3 https://readiness-probe-scale-test.s3.amazonaws.com/readiness -o /var/lib/mongodb-mms-automation/probes/readinessprobe \
```

Comment:
Will update this again once I've moved the readiness probe to the correct bucket